resources/lang/en/settings.php

```diff
@@ -5,5 +5,8 @@
     'updated' => 'Settings successfully saved! If you changed your email address you\'ll receive an email address to re-confirm it.',
     'deleted' => 'Account was successfully removed.',
     'password.updated' => 'Password successfully changed!',
-
+    'api_token' => [
+        'created' => 'API token created! Please copy the following token as it will not be shown again: :token',
```

Comment:
I haven't pulled this down to look yet (will try tomorrow or over the weekend), but does the token need to be on a new line or have some space from the rest of the flash message?

Reply:
@joedixon FYI I'm already working on this locally.